Modix.Data/Models/Moderation/NoteEntity.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Modix.Data.Models.Moderation
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    public class NoteEntity
+    {
+        [Key, Required, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        public ulong UserId { get; set; }
```

Comment:
NpgSQL doesn't support `ulong` (per se). Entity classes need to use `long` where applicable, and cast back and forth with `ulong` in projections and other model translation operation. NpgSQL does support a native 64-bit int type, so the binary representation will remain intact the whole way through.